src/components/NurseryWithdrawals/PlantingProgressMap.tsx

```diff
@@ -1,7 +1,33 @@
+import { useAppSelector } from 'src/redux/store';
+import { selectPlantingSite } from 'src/redux/features/tracking/trackingSelectors';
+import { MapService } from 'src/services';
+import { MapData } from 'src/types/Map';
+import { useEffect, useState } from 'react';
```

Comment:
Not requesting any changes. Just something that I hope we can fix soon.

We don't have a strict rule for ordering imports yet (created a ticket for that).. i usually order them as such
1. react
2. external libraries (eg. mui)
3. internal libraries (eg. web-components)
4. internal models/constants
5. internal services/utils/functions
7. internal components

There are node packages that can also order them, we could pick one maybe. Just wanted to put this on everyone's radar in case we have preferences.

Reply:
Good point, I've just let them pile up however. It would be nice to get something standardized, I agree

Reply:
Cool,  I can request bumping up that ticket in the tech_investment pile.